File: bench/type4/adversarial/cases/collection_empty_check/positive.rs
fn rust_len_empty(items: Vec<i32>, other: Vec<i32>) -> bool {
    items.len() == 0
}

fn rust_named_empty(values: Vec<i32>, other: Vec<i32>) -> bool {
    values.is_empty()
}

fn rust_len_nonempty(items: Vec<i32>, other: Vec<i32>) -> bool {
    items.len() != 0
}

fn rust_named_nonempty(values: Vec<i32>, other: Vec<i32>) -> bool {
    !values.is_empty()
}

File: bench/type4/adversarial/cases/collection_empty_check/positive.swift
func swiftCountEmpty(_ items: [Int], _ other: [Int]) -> Bool {
    return items.count == 0
}

func swiftNamedEmpty(_ values: [Int], _ other: [Int]) -> Bool {
    return values.isEmpty
}

func swiftCountNonempty(_ items: [Int], _ other: [Int]) -> Bool {
    return items.count != 0
}

func swiftNamedNonempty(_ values: [Int], _ other: [Int]) -> Bool {
    return !values.isEmpty
}
